### ISSUE TITLE
IntelliSense between c++98 and c++03 is different

### ISSUE THREAD

**Issue by ghuser404** [microsoft/vscode-cpptools#5225]
While testing #5220, I noticed that if I specify `"cppStandard": "c++98"` in c_cpp_properties.json, I do not get red squiggles for code below, but if I specify `"cppStandard": "c++03"`, then I do. That is without CMake Tools extension.

I also do get red squiggles when I use CMake Tools extension as configuration provider `"configurationProvider": "ms-vscode.cmake-tools"` and having `target_compile_features(test PRIVATE cxx_std_98)` in CMakeLists.txt.

The error is `error: '>>' should be '> >' within a nested template argument list`.
```
#include <vector>

int main()
{
    std::vector<std::vector<int>> v;
    return 0;
}
```

**Comment by sean-mcmanus** [comment]
`error: '>>' should be '> >' within a nested template argument list` is from gcc and not our IntelliSense, but it appears our IntelliSense is not configured correctly when c++98 is used. The workaround is to use c++03.

**Comment by ghuser404** [comment]
Yeah, this error is from gcc. With c++03 cpptools gives `space required between adjacent '>' delimiters of nested template argument lists ('>>' is the right shift operator)` - but you get the idea.

**Comment by sean-mcmanus** [comment]
Fixed with https://github.com/microsoft/vscode-cpptools/releases/tag/0.28.0-insiders .

**Comment by ghuser404** [comment]
This now works correctly. However, like I mentioned in #5157, I get `Standard Version: c++03` with either `"cppStandard": "c++98"` or `"compilerArgs": ["-std=c++98"]`. Not sure if this is a problem?

**Comment by sean-mcmanus** [comment]
Yeah, that is correct -- our back-end parser treats C++98 and C++03 identically (I think they're very similar). Previously, using "c++98" was causing some later standard to be used.

**Comment by elahehrashedi** [comment]
This issue is fixed in 0.28.0.
0.28.0 release: https://github.com/microsoft/vscode-cpptools/releases/tag/0.28.0
